### DISCUSSION TITLE
How to get the selected item of v-list in vuetify3

### DISCUSSION THREAD

**Discussion by Kugeleis** [vuetifyjs/vuetify#16400] [Q&A] [answered]
In vuetify2 there was the v-list-item-group offering the v-model prop. How is this done in vuetify3?
```
<v-list>
    <v-item-group v-model="value"
      ><v-list-item v-for="item in items" :key="item">
          <v-list-item-title>{{ item }}</v-list-item-title>
        </v-list-item>
    </v-item-group>
</v-list>
```
doesn't work.

**Comment by santiagodeheza** [comment, marked as answer]
You only need `v-model:selected="..." ` in v3
[Playground reproduction:](https://play.vuetifyjs.com/#eNptVMtu2zAQ/JUFL04AyXYcFAUExUBQ9NzCyc3OgaVWDRFKJEhKdSDo37vUK5KSi7GvGQ6XY50b5qzYPRqzrStkCUs9FkZxj8dLCZDWMTemC7uk4LIcMsqFzvAIT6hQeMxAEjSBpgE3VtoW0l03NWHqWHCbgVDcuYcLK64xr7y+MCj4Nf4nM/9K1fv9/sImTIdS0nkgAUSmkvEAGh1DYsiwdNK/U1HownDhFxwTS+yqP6/IM7TH08/fv07PT+nuU+crXLjfvA6kJ9eWzrsJrQjkLciyW4O7sOVk8oZBmPxUr7mqMHQC+aoptOrojZUFt+/L9kIiiRzfjUQ5pX1iLBoss9VYdxspdAlJ+B0O3oaYthUWEcI1925milVrtpvYS6/C+R6vfmQO8cC8GlzueN6fv/xQ/zAQFYKDRk9SOpkyJL1dV4pTJ6w0nozpq97NsjDaemjAYg4t5FYXsKF/wKa/IO2A7Db5+CGM3Zz3L7cfze6Zhw6cezUNyCzZRxAuncDmhFzFz7LADXmDUFQqMhkLpcXbBtpoBrqbQI9VJrEUKwwXQlelX6EOE+qHLmu0TpK0JTBX/O+ICvLTXb8L2gJrI3a//bY9HFgI7vbb7yzKuXIYLT4Kfe3lP2naQqM=)


```html
<template>
  <v-app>
    <v-main>
      <code> {{ selected }} </code>   
      <v-card class="mx-auto" max-width="300">

        <v-list v-model:selected="selected" density="compact">
          <v-list-subheader>REPORTS</v-list-subheader>
          <v-list-item
           v-for="(item, i) in items"
           :key="i"
           :value="item"
           active-color="primary"
           >

            <template v-slot:prepend>
              <v-icon :icon="item.icon"></v-icon>
            </template>

            <v-list-item-title v-text="item.text"></v-list-item-title>
          </v-list-item>
        </v-list>
      </v-card>   
    </v-main>
  </v-app>
</template>

<script setup>
  import { ref } from 'vue'

  const selected = ref([])
  const items = ref( [
    { text: 'Real-Time', icon: 'mdi-clock' },
    { text: 'Audience', icon: 'mdi-account' },
    { text: 'Conversions', icon: 'mdi-flag' },
  ])
</script>
```

**Reply by protsaenko** [reply]
Thank you for example, but is there a way to select multiple items, as it was with [`v-list-item-group`](https://v2.vuetifyjs.com/en/api/v-list-item-group/#props)?

**Reply by PrimozGodec** [reply]
@santiagoaloi
 Is there a way to do a single-item selection? With this snippet, I can only select multiple items. I tried to set it to `select-strategy="single"`, but it is not working.

**Reply by santiagodeheza** [reply]
Hi, I’m not entirely sure what you mean. This list isnt confiugured for multiple selections, it only pushes the selected object into the array.
Do you mean that by “single item” you only want one property from the object added to the array?

Also, note that "single" isn’t a valid value for the select-strategy prop.

**Reply by PrimozGodec** [reply]
Thank you for the answer. Exactly — I was looking for a mechanism that allows adding only one element to the list, and automatically removes the previous one when a new item is selected. I can handle this with a custom script function, but I was hoping for a cleaner solution.

Is there another component that renders similarly to a list but is better suited for single-item selection?

**Reply by santiagodeheza** [reply]
Allright, then you only need to tweak  the value prop in `v-list-item` - [take a look at this reproduction](https://play.vuetifyjs.com/#eNptVE1vozAQ/SsjX9JKkKSpVishGqla7XlXaW9JD157srVqsGWbNBHiv3cMgQLtBc3Xm3kzPNjXzDuxerR2eaqQZSwPWFjNA24PJUB+Srm1rdk6BVfl1SNfGIlbeEKNIqAERVBQMoO6Bt8HmwbyVVs4wE6p4E6C0Nz7hwMrzimvgjkwKPg5fVcyvFL0fr0+sAHTorTygWpKyYNxFyA21FZn/SgC9Sb1klh6FS4UFKawXIRJt6Ff6qt/r8gluu3u998/u+enfPUl8x0uLjuOA/E5GkfzbmIqAXULqmxv4g9sWpm9YSSmvsRPXFcYM4RaKlpjmhdGtxOsUwV3l2l6wpJ49u+ReHltQmYdWizlrKxdSAlTQhafw2yy6WDxFtGc916NRDJLjc6TBhV0nB/wHPrO0b52nhVOzzzOj2VwjX+qiQJRTr1GyR1EGp1OvjPGuRdO2UAqDVWnblVY4wLU4PAIDRydKWBBX8SiW5BuQNobRP0Qy27265fbz2T7pq8Z2Hds6vg5rBOIS2ew2CHX6bMqcEHyIBSFCqlSoY14W0CTjEB3A+ixkgpLMcNwIUxVhhlqM6B+mfKEziuiNgUeNf/foyL9fNXdgq7AmoTdL38sNxsWjbv18idLjlx7TCY/iS728gFNokh6)

**Reply by KaelWD** [reply]
The default is `select-strategy="single-leaf"`. single-leaf and single-independant will both behave the same if you don't have nested items.

**Comment by Kugeleis** [comment]
Thanks. It would be great if this could be explained in the docs.
